projects/demo-maps/src/app/route-components/route-example-cesium/route-example-cesium.component.ts

```diff
@@ -338,5 +327,21 @@ export class RouteExampleCesiumComponent implements OnInit, OnDestroy {
 
     }
   }
+  setViewAngle() {
+    /** set map rotation with the MapStateService */
+    this.mapStateSvc.setViewAngle(45);
+  }
+  resetViewAngle() {
+    /** set map rotation with the MapStateService */
+    this.mapStateSvc.setViewAngle(0.01);
```

Comment:
What are the low values "0.0.1" good for?

Reply:
When setting the value to 0, there seems to be an rounding issue with the conversions. So it is more stable to use very small numbers instead of 0. Internally, small numbers close to 0 are then snapped to zero (https://openlayers.org/en/latest/apidoc/module-ol_View-View.html): 

> The rotation constraint snaps to specific angles. It is determined by the following options: enableRotation and constrainRotation. By default rotation is allowed and its value is snapped to zero when approaching the horizontal.

Reply:
Then I would handle that internally, but leave the interface as a user would expect.

e.g.
```
let radAngle;
if(viewAngle === 0){
  radAngle = viewAngle;
}else{
  radAngle = CesiumMath.toRadians(viewAngle-90)
}
```
- https://github.com/CesiumGS/cesium/blob/1.114/packages/engine/Source/Core/Math.js#L441
- https://github.com/CesiumGS/cesium/blob/1.114/packages/engine/Source/Core/defined.js


and Ol
- https://github.com/openlayers/openlayers/blob/84e690734799df9b96ba22e799e2fe9b15c00cb2/changelog/v3.2.0.md?plain=1#L11